guava-recent-unit-tests/src/test/java/com/uber/nullaway/guava/NullAwayGuavaParametricNullnessTests.java

```diff
@@ -125,12 +126,23 @@ public void testFunctionMethodOverride() {
             "Test.java",
             "package com.uber;",
             "import com.google.common.base.Function;",
+            "import com.google.common.collect.Maps;",
+            "import java.util.Map;",
```

Comment:
Why these new imports?

Reply:
Removing. Left-over from when this test code was all in the same test.